src/sage/matroids/basis_matroid.pyx

```diff
@@ -515,42 +520,46 @@ cdef class BasisMatroid(BasisExchangeMatroid):
         cdef frozenset se = frozenset([e])
         return BasisMatroid(groundset=self._E + (e,), bases=[B | se for B in self.bases()])
 
-    cpdef relabel(self, l):
-        """
+    cpdef relabel(self, f):
+        r"""
         Return an isomorphic matroid with relabeled groundset.
 
-        The output is obtained by relabeling each element ``e`` by ``l[e]``,
-        where ``l`` is a given injective map. If ``e not in l`` then the
+        The output is obtained by relabeling each element ``e`` by ``f[e]``,
+        where ``f`` is a given injective map. If ``e not in f`` then the
```

Comment:
Best to rewrite this without the `in` operator, for example as "if ``f[e]`` is not defined..."
because of the Sage-specific `Family` classes, which are like `dict` or other `Mapping`s but `in` refers to values, not keys.